src/pages/patterns/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Patterns overview
+description: >
+  Existing design solutions for a more consistent UX
```

Comment:
@ctancinco, @allisonacs for all other pillars (Content, Design, Components) this description matches the home page 1:1, including no-period. Should the same thing happen here or on the home page?

Reply:
Yes. Let's update the homepage to match this copy, it feels more on brand.